velox/docs/functions/presto/datetime.rst

```diff
@@ -1,7 +1,79 @@
 =====================================
-Date and Time Functions
+Date and Time Functions and Operators
 =====================================
 
+Date and Time Operators
+-----------------------
+
+.. list-table::
+   :widths: 15 60 25
+   :header-rows: 1
+
+   * - Operator
+     - Example
+     - Result
+   * - ``+``
+     - ``interval '1' second + interval '1' hour``
+     - ``0 01:00:01.000``
+   * - ``-``
+     - ``interval '1' hour - interval '1' second``
+     - ``0 00:59:59.000``
+   * - ``*``
+     - ``interval '1' second * 2``
+     - ``0 00:00:02.000``
+   * - ``*``
+     - ``2 * interval '1' second``
+     - ``0 00:00:02.000``
+   * - ``*``
+     - ``interval '1' second * 0.001``
+     - ``0 00:00:00.001``
+   * - ``*``
+     - ``0.001 * interval '1' second``
+     - ``0 00:00:00.001``
+   * - ``/``
+     - ``interval '15' second / 1.5``
+     - ``0 00:00:10.000``
+
+.. function:: plus(x, y) -> [same as x]
+
+    Returns the sum of ``x`` and ``y``. ``x`` and ``y`` are both intervals day
+    to second. Returns ``-106751991167 07:12:55.808`` when the addition
+    overflows in positive. Returns ``106751991167 07:12:55.807`` when the
+    addition overflows in negative.
+
+.. function:: minus(x, y) -> [same as x]
+
+    Returns the result of subtracting ``y`` from ``x``. ``x`` and ``y`` are
+    both intervals day to second. Returns ``-106751991167 07:12:55.808`` when
+    the subtraction overflows in positive. Returns ``106751991167 07:12:55.807``
```

Comment:
To make sure this is right  when subtraction overflow is positive we return -tive value and vice versa ?

Reply:
Hi @kgpai, yes, I double checked in Presto-Java. Try the following example:
```
select interval '1' second * INFINITY() + interval '1' second --> -106751991167 07:12:54.809
```
The first operand of + is the maximum positive interval.